content/en/docs/Installation/dashboard.md

```diff
@@ -0,0 +1,315 @@
+<!--
+---
+title: "Install Tekton Dashboard"
+linkTitle: "Tekton Dashboard"
```

Comment:
For the component specific installation pages, is it possible to have the pages in the projects themselves and pull the content from there? Otherwise, anytime a component's installation process changes , we'd have to update the website page separately.

Reply:
Hmm... I recently submitted a [PR to upstream Docsy](https://github.com/google/docsy/pull/995) that adds a shorcode to include the contents of one file into another. We can use that to have the Installation content under the same section while keeping it in the repo.

Thanks for the suggestions. I'll look into that.

Reply:
Installation instructions must be versioned, same as the reference docs, so the instructions shown here must change when a different version is picked from the dropdown.

It may be possible to handle that through the sync script, but if we can get more native support in docsy that's even better.

I think the main disconnect between Tekton and Docsy is that Tekton has multiple components released independently, and I don't think Docsy natively supports that?

Reply:
Docsy supports versioning but for the entire docset, not for individual files or folders. But we already have a shortcode for the synced docs, we can make something similar for the installation.

> It may be possible to handle that through the sync script

I was going through the script to figure out the best way to do it. I think it should be possible with the current functions. I'll close this PR and come up with a solution after I'm done with a few other PRs I'm currently working on.